_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -127,7 +130,7 @@ Full length autocomplete terms.
 
 #### Passed Example 8
 
-The `input` element does not have a semantic role that is a widget role, but still participates in sequential focus navigation, and has a single autocomplete term.
+This `input` element has an [explicit role][] of `none`. However, it is [focusable][] (by default). Thus it has a [semantic role][] of `textbox` due to [Presentational Roles Conflict Resolution][]. It has a single autocomplete term.
```

Comment:
This example does not have a label. Should we be adding it?

Reply:
This is topic for another PR, and much more generic than this specific rule.
See #1215.